src/components/containers/popover/PopoverLink.js

```diff
@@ -65,7 +67,8 @@ class PopoverLink extends React.Component {
           }}
         >
           <PopoverButton
-            aria-haspopup="dialog"
+            aria-haspopup="true"
```

Comment:
The [aria spec](https://www.w3.org/TR/wai-aria/states_and_properties#aria-haspopup) there is no "dialog" value for haspopup, only true/false. There is a [dialog role](https://www.w3.org/TR/wai-aria/roles#dialog) though. Is that what you're thinking about, maybe?